spec/system/consumer/checkout/payment_spec.rb

```diff
@@ -250,6 +250,25 @@
               click_on "Next - Order summary"
               proceed_to_summary
             end
+
+            context "when saving card" do
+              it "selects Stripe SCA and proceeds to the summary step" do
+                stub_customers_post_request(email: order.user.email)
+                stub_payment_method_attach_request
```

Comment:
Hum, interesting! You're using Stripe stubs on a system spec :eyes: 
I think I've seen this before, but I have not gotten into it yet.

I wonder if instead of these stubs we could not use VCR (as for example, on [this spec](https://github.com/openfoodfoundation/openfoodnetwork/commit/baef7de663bcc2d657d9d7063186b3dbe3566a8a))

In any case, not a blocker - just wondering.

Reply:
I should have added a comment, from the commit message : 
> I used stripe stubs for the new scenario because storing a card on
> stripe depends on some client side interaction with Stripe. We can't
> capture that with VCR.

That's my understanding at least, I might have missed something